javascript/ql/src/semmle/javascript/PackageExports.qll

```diff
@@ -42,7 +42,7 @@ PackageJSON getTopmostPackageJSON() {
  * Gets a value exported by the main module from one of the topmost `package.json` files (see `getTopmostPackageJSON`).
  * The value is either directly the `module.exports` value, a nested property of `module.exports`, or a method on an exported class.
  */
-private DataFlow::Node getAValueExportedByPackage() {
+DataFlow::Node getAValueExportedByPackage() {
```

Comment:
```suggestion
private DataFlow::Node getAValueExportedByPackage() {
```
Do we have an uncommitted test file that needs this to be public?

Reply:
No. I just accidentally committed something I shouldn't have.